app/lib/actions/katello/content_view/publish.rb

```diff
@@ -32,19 +33,29 @@ def plan(content_view, description = "", options = {})
                                                           :triggered_by => options[:triggered_by]
                                                          )
           source_repositories = []
-          content_view.publish_repositories do |repositories|
+          content_view.publish_repositories(options[:override_cvvs]) do |repositories|
```

Comment:
can we rename 'override_cvvs' to 'override_component_versions'  I think its a bit more clear